_posts/2015-08-10-type-driven-development.html

```diff
@@ -74,7 +74,7 @@
     When in the <em>Ready</em> or <em>No message</em> states, the Polling Consumer may also realise that time is running out, and so decide to quit, in which case the (final) state is the <em>End</em> state.
   </p>
   <p>
-    This means that the Polling Consumer needs to measure what it does, and those measurements influence what it decides to do.
+    This means that the Polling Consumer needs to measure what it does, and have those measurements influence what it decides to do.
```

Comment:
Here I was attempting to bring out the fact that you felt that there was a need to time things, and that those timings would serve as inputs to that influencing the decision. When I read the initial form, that linkage is not there; it's as if the act implied "measure" directly "influence"s the decision.

Your proposal above retains the same weakness in my view hence I wouldn't see it as a meaningful improvement.